pkg/configutils/kots.go

```diff
@@ -0,0 +1,35 @@
+package configutils
+
+import (
+	"fmt"
+	"os"
+	"sigs.k8s.io/yaml"
+)
+
+type gvk struct {
+	ApiVersion string `yaml:"apiVersion"`
+	Kind       string `yaml:"kind"`
+}
+
+// ValidateKotsConfigValues checks if the file exists and has the 'kots.io/v1beta1 ConfigValues' GVK
+func ValidateKotsConfigValues(filename string) error {
```

Comment:
Is there a function in KOTS that does this?

Reply:
we can probably make it a hidden kots cli command